Use always-malloc by default. The slice allocator is only there because windows' malloc is slow.

Ref: https://gitlab.gnome.org/GNOME/glib/-/issues/1079
Ref: https://gitlab.com/qemu-project/qemu/-/issues/285

diff --git a/glib/gslice.c b/glib/gslice.c
index 4e999202e..f4b8051c3 100644
--- a/glib/gslice.c
+++ b/glib/gslice.c
@@ -283,7 +283,7 @@ static GPrivate    private_thread_memory = G_PRIVATE_INIT (private_thread_memory
 static gsize       sys_page_size = 0;
 static Allocator   allocator[1] = { { 0, }, };
 static SliceConfig slice_config = {
-  FALSE,        /* always_malloc */
+  TRUE,        /* always_malloc */
   FALSE,        /* bypass_magazines */
   FALSE,        /* debug_blocks */
   15 * 1000,    /* working_set_msecs */
